gnmpagerduty/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models import Model,BigIntegerField,IntegerField,CharField,DateTimeField
+
+
+class StorageData(Model):
+    storage_id = CharField(max_length=64,unique=True)
+    maximum_size = BigIntegerField(blank=True,null=True)
+    current_size = BigIntegerField(blank=True,null=True)
+    trigger_size = BigIntegerField(blank=True,null=True)
+    incident_key = CharField(max_length=64,blank=True, default='')
```

Comment:
I just did not bother taking it out of the first model.

What is a foreign key?